package.json

```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "build": "tsc",
     "checksize": "rollup -c ; cat lit-html.bundled.js | gzip -9 | wc -c ; rm lit-html.bundled.js",
-    "gen-docs": "typedoc --readme ./docs/_api-readme.md --mode file --ignoreCompilerErrors --excludeNotExported --excludePrivate --out ./docs/api --gaID UA-39334307-23 ./src/lit-html.ts ./src/lib/*.ts ./src/directives/*.ts",
```

Comment:
Maybe we should move this script to docs/package.json

Reply:
I encountered some problems when I tried to do so before, but I think I've figured them out. Do you know if anything else relies on this gen-docs script, or can I remove it from the top-level package.json entirely?